samples/subsys/logging/kernel_event_logger/README.rst

```diff
@@ -0,0 +1,40 @@
+.. _kerneleventlogger_sample:
```

Comment:
This is the first document in the logging area, so you'll need to also add a file samples/subsys/logging/logging.rst that looks like samples/subsys/console/console.rst  (copy that one and edit the title).  This will remove the "not included in a toc-tree" warning.

Reply:
ok